nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -262,7 +284,7 @@ def _add_statistic_point(self, container: StatisticPointsContainer, point: Targe
         )
 
     def _create_input_data(
```

Comment:
I couldn't find where you use this function after your changes. Maybe you can delete it safely.